lib/theme_check/checks/translation_key_exists.rb

```diff
@@ -23,8 +38,9 @@ def on_variable(node)
 
     private
 
-    def key_exists?(key)
-      pointer = @theme.default_locale_json.content
+    def key_exists?(key, hash)
```

Comment:
Maybe a name to clarify its use:

```suggestion
    def key_exists?(key, default_locale)
```

Reply:
Yeah, we don't need this anymore. I did initially when the file was also a hash. Opted to go with an array of keys instead (this way we only need the keys and not the content). 

I'll remove this change.